src/assets/images/index.ts

```diff
@@ -1,8 +1,7 @@
-import AicsLogoSVGAsset from "./AICS-logo.svg?react";
 import AicsLogoAndNameSVGAsset from "./AICS-logo-and-name.svg?react";
+import AicsLogoSVGAsset from "./AICS-logo.svg?react";
```

Comment:
static analysis wants to sort these imports.

Reply:
The girls* are fighting
(*eslint and prettier)